### PR TITLE
bit 14.2.0

### PR DESCRIPTION
**Pull request by amitgilad3**
bit has moved to binary format so i have created a new formula to support this

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bit.rb [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
+  desc "Easily share code between projects with your team"
+  homepage "https://bit.dev"
+  url "https://github.com/teambit/bit/releases/download/v14.2.0/bit-bin-macos"
+  version "14.2.0"
+  sha256 "fe3e125bee0139364c48c1d404c78132ed69d4287d02a92bb73cb4c45f42ca2f"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bin.install "bit-bin-macos"
```
Comment (chenrui333) [inline Formula/bit.rb]: it looks like this formula is not built from the source?

Comment (amitgilad3) [reply Formula/bit.rb]: you are correct - we now create binary distributions for darwin, linux, windows

Comment (fxcoudert) [reply Formula/bit.rb]: Homebrew core formulas must build from source: https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md

**Comment by chenrui333** [comment]
Created a PR for updating the version to the latest, `v14.2.2`

**Comment by chenrui333** [comment]
https://github.com/Homebrew/homebrew-core/pull/42395

**Comment by SMillerDev** [comment]
Homebrew formula must build from source. This version is not suitable for core

**Pull request closed without merging**